tests/test_selenium.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver import Firefox
+from dotenv import find_dotenv, load_dotenv
+from selenium.webdriver.common.keys import Keys
+from todo_app.app import create_app
+from todo_app.trello_cards import create_a_board, delete_a_board
+from selenium.webdriver.chrome.options import Options
+
+
+
+@pytest.fixture(scope="module")
+def driver():
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+    with webdriver.Firefox() as driver:
+        yield driver
+
+
+
+@pytest.fixture(scope='module')
+def test_app():
+    # Create the new board & update the board id environment variable
+    board_id = create_a_board('testboard')
+    os.environ['TRELLO_BOARD_ID'] = board_id
```

Comment:
Elsewhere in your code you refer to this as `BOARD_ID` so your app is still using your default board currently